subset/security/tlsfaux/server.py

```diff
@@ -28,22 +28,23 @@
 #OP_NO_TLSv1_2
 #OP_NO_TLSv1_3
 
-def do_something(connstream,data):
-	print "do_something:",data
+def parse_data(connstream,data):
```

Comment:
How is this parsing data? What's the purpose of this function?  It's more than "something" but less than "parse".